LICENSE

```diff
@@ -1,2 +1,30 @@
 # TODO: Write license file.
```

Comment:
The todo should be removed from the final version

Reply:
yeah, eventually.